tests/testfiles/config/sources.php

```diff
@@ -1,16 +1,64 @@
 <?php
 
-$priorities = [
+$mozilla = ['en-US', 'fr', 'it'];
+$slogans_locales = ['fr', 'it'];
+
+$repo0_langfiles = [
     'file1.lang' => [
-        'critical' => ['all'],
-        'obsolete' => ['fr'],
+        'supported_locales' => ['fr'],
+        'deadline'          => '2017-01-30',
+        'flags'             => [
+            'obsolete' => ['fr'],
+        ],
+        'priority' => [
+            1 => ['fr'],
+        ],
     ],
     'file2.lang' => [
-        'critical'  => ['fr'],
-        'obsolete'  => ['all'],
-        'testflag1' => ['all'],
-        'testflag2' => ['fr'],
+        'flags' => [
+            'obsolete'  => ['all'],
+            'testflag1' => ['all'],
+            'testflag2' => ['fr'],
+        ],
+        'priority' => [
```

Comment:
It can't break, because the code exits as soon as it finds a perfect match, while it keeps checking if it finds 'all' (that's actually what I'm testing). 

Having said that, having another file that tests that case wouldn't hurt.